common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -54,6 +54,14 @@ def create_modulestore_instance(
     )
 
 
+def mock_tab_from_json(tab_dict):
```

Comment:
Where is `CourseTab.from_json` tested? If tests are passing when this just returns the `tab_dict`, it seems like this behavior is not being verified.